### PR TITLE
Add tests for high_needs? in schools

### PR DESCRIPTION
**Pull request by jopolsky**
Adding the tests for high_needs?

Related to: https://github.com/code-dot-org/code-dot-org/pull/18853

### CODE REVIEW

File: dashboard/test/models/school_test.rb [hunk 1]
```diff
@@ -21,4 +21,76 @@ class SchoolTest < ActiveSupport::TestCase
       }
     )
   end
+
+  test 'high needs false when no stats data' do
+    school = School.find_by_id('20000100206')
+    assert_equal(false, school.high_needs?)
```
Comment (aoby) [inline dashboard/test/models/school_test.rb]: nit: `refute school.high_needs?`

Comment (aoby) [reply dashboard/test/models/school_test.rb]: Same elsewhere

File: dashboard/test/models/school_test.rb [hunk 2]
```diff
@@ -21,4 +21,76 @@ class SchoolTest < ActiveSupport::TestCase
       }
     )
   end
+
+  test 'high needs false when no stats data' do
+    school = School.find_by_id('20000100206')
```
Comment (aoby) [inline dashboard/test/models/school_test.rb]: Do we need this specific school (by id)? Can we just get the first, or create one on the fly (via FactoryGirl's `create :school`)?

File: dashboard/test/models/school_test.rb (hunk 1 above)
Comment (jopolsky) [reply dashboard/test/models/school_test.rb]: upated

File: dashboard/test/models/school_test.rb (hunk 2 above)
Comment (jopolsky) [reply dashboard/test/models/school_test.rb]: sure, updated

**Pull request merged**